### DISCUSSION TITLE
Rack Elevations default views

### DISCUSSION THREAD

**Discussion by tophe75** [netbox-community/netbox#9565] [Ideas]
A really nice feature would be to change the default view behavior so that 'Front' view is 'Normal' (you se the racks as they are physically in the data center, but when you switch to 'Rear' view the default in 'Reversed' .

A switch button might be the best choice to change this setting.

**Comment by jemery27** [comment]
I second this - the rear elevation should be backward.

ok - nevermind I see a sort field now - doh
